[BUG] Error executing DDL alter table drop constraint via JDBC Statement

- GenerationTarget encountered exception accepting command : Error executing DDL "alter table img_post_comments drop constraint FKbf4rwnww2l9fd40lmv8ldjncv" via JDBC Statement
- Caused by: org.postgresql.util.PSQLException: ERROR: constraint "fkbf4rwnww2l9fd40lmv8ldjncv" of relation "img_post_comments" does not exist

--- a/src/main/java/com/amplifier/config/AppConfig.java
+++ b/src/main/java/com/amplifier/config/AppConfig.java
@@ -5,7 +5,7 @@ import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.EnableAspectJAutoProxy;
 
 @Configuration
-@ComponentScan(basePackages = { "com.amplifier.repos", "com.amplifier.services", "com.amplifier.aspects" })
+@ComponentScan(basePackages = { "com.amplifier.repositories", "com.amplifier.services", "com.amplifier.aspects" })
 @EnableAspectJAutoProxy(proxyTargetClass = true)
 public class AppConfig {
 

--- a/src/main/java/com/amplifier/config/Swagger2Configuration.java
+++ b/src/main/java/com/amplifier/config/Swagger2Configuration.java
@@ -19,7 +19,7 @@ public class Swagger2Configuration {
     public Docket swaggerConfiguration() {
         return new Docket(DocumentationType.SWAGGER_2)
                 .select()
-                .paths(PathSelectors.ant("/api/*"))
+                .paths(PathSelectors.ant("/api/v1/*"))
                 .apis(RequestHandlerSelectors.basePackage("com.amplifier"))
                 .build()
                 .enableUrlTemplating(true)

